src/Compilers/VisualBasic/Portable/Symbols/Metadata/PE/SymbolFactory.vb

```diff
@@ -49,6 +49,11 @@ Namespace Microsoft.CodeAnalysis.VisualBasic.Symbols.Metadata.PE
             Return False
         End Function
 
+        Friend Overrides Function IsConstModifierType(moduleSymbol As PEModuleSymbol, type As TypeSymbol) As Boolean
+            ' VB doesn't deal with ref-readonly parameters or return-types.
+            Return False
```

Comment:
This means that if VB `MetadataDecoder` tries to read a symbol with a modreq of type `IsConst`, it will return `False` meaning it doesn't support it, and the symbol will be marked appropriately. Check `IsVolatileModifierType()` in the same file.

---
In reply to: [117828980](https://github.com/dotnet/roslyn/pull/19658#discussion_r117828980) [](ancestors = 117828980)